code/_onclick/hud/blob_overmind.dm

```diff
@@ -126,7 +126,7 @@
 /obj/screen/blob/Split
 	icon_state = "ui_split"
 	name = "Split consciousness (100)"
-	desc = "Creates another Blob Overmind at the nearest node. One use only.<br>Offsprings are be unable to use this ability."
+	desc = "Creates another Blob Overmind at the targetted node. One use only.<br>Offsprings are unable to use this ability."
```

Comment:
"targeted" only has one t at the end

Technically "Offspring" doesn't need the s to be plural either

Reply:
Fixed.